docs/creating_your_tooltip.md

```diff
@@ -2,129 +2,125 @@
+If you use [`yarn`](https://yarnpkg.com/), you can install the library via:
 
 ```bash
-npm install vega-tooltip
+yarn add vega-tooltip
 ```
 
-The tooltip library contains a `js` file and a `css` file in `build` directory:
+The tooltip library has the compiled tooltip library in the `build` directory:
 
 ```
 vega-tooltip/build/vega-tooltip.js
-vega-tooltip/build/vega-tooltip.css
 ```
 
-Alternatively, you can get vega-tooltip via [`bower`](https://bower.io/):
-```bash
-bower install vega-tooltip
-```
+If you want to manually include the library and its dependencies, you can add the following lines to your HTML script tag. Vega Tooltip works with [`vega`](https://vega.github.io/vega/) and [`vega-lite`](https://vega.github.io/vega-lite/).
 
-If you want to manually include the library and its dependencies, you can add the following lines to your HTML script tag. Vega Tooltip depends on either [`vega`](https://vega.github.io/vega/) and [`vega-lite`](https://vega.github.io/vega-lite/) depending on which one you want to include tooltip.
-<br>
 ```html
-<script src="https://cdn.jsdelivr.net/npm/vega"></script>
-<script src="https://cdn.jsdelivr.net/npm/vega-lite"></script>
+<script src="https://cdn.jsdelivr.net/npm/vega@3"></script>
+<script src="https://cdn.jsdelivr.net/npm/vega-lite@2"></script>
 
 <!-- Vega Tooltip -->
 <script src="https://cdn.jsdelivr.net/npm/vega-tooltip"></script>
-<link rel="stylesheet" type="text/css" href="https://cdn.jsdelivr.net/npm/vega-tooltip/build/vega-tooltip.min.css">
 ```
 >Note that if you use [Vega-Embed](https://github.com/vega/vega-embed/) to deploy your visualization, you need to include it as a dependency
 
 ```html
 <!-- vega-embed -->
-<script src="https://cdn.jsdelivr.net/npm/vega-embed"></script>
+<script src="https://cdn.jsdelivr.net/npm/vega-embed@3"></script>
+```
+
+Instead of including Vega Tooltip as a script, you can import it in your bundle. 
+
+```js
+import vegaTooltip from 'vega-tooltip';
 ```
 
 ## Step 2. Create A Visualization
 
 In your HTML `<body>`, create a placeholder for your visualization. Give the placeholder a unique `id`, to which you can later refer in your JavaScript. For example:
 
 ```html
-<!-- Placeholder for my scatter plot -->
-<div id="vis-scatter"></div>
+<!-- Placeholder for my plot -->
+<div id="vis"></div>
 ```
 
-For [Vega-Lite](https://vega.github.io/vega-lite/):
-
-You can create your visualization using [`vegaEmbed`](https://github.com/vega/vega-embed). The following JavaScript code refers to the visualization placeholder by id selector (`#vis-scatter`).
+You can create your visualization using [`vegaEmbed`](https://github.com/vega/vega-embed). The following JavaScript code refers to the visualization placeholder by id selector (`#vis`). The spec can be a Vega or Vega-Lite specification.
 
 ```js
-var opt = {
-  mode: "vega-lite",
-};
-vegaEmbed("#vis-scatter", vlSpec, opt)
+vegaEmbed("#vis", spec)
   .then(function(result) {
```

Comment:
might as well use arrow function here... but up to you

Reply:
We use es5 in our docs